lib/active_setting/loader.rb

```diff
@@ -6,14 +6,27 @@ def self.load_settings(filename = self.config_filename)
       settings_config(filename).each do |category_name, settings|
         settings.each do |setting_name, values|
           Setting.register(setting_name.to_sym, values.merge(
-            :data_type => values['type'],
-            :category => category_name,
-            :name => setting_name
+            data_type: values['type'],
+            category:  category_name,
+            name:      setting_name
           ))
         end
       end
     end
 
+    def self.build_hash(filename = self.config_filename)
+      settings_config(filename).map.with_object({}) do |(category_name, settings), hash|
+        settings.each do |setting_name, values|
+          attrs = values.merge(
+            data_type: values['type'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.